pyvelox/test/test_vector.py

```diff
@@ -156,3 +158,32 @@ def test_append(self):
 
         with self.assertRaises(TypeError):
             ints2.append(strs2)
+
+    def test_export_to_arrow(self):
+        vector = pv.from_list([1, 2, 3])
+        vector_ptr = pv.export_to_arrow(vector)
```

Comment:
Is it common to have c style names like _ptr etc ?

Reply:
Are you asking about using the `_ptr` here, or the `snake_case` style used? For the `_ptr` usage, I can use better names, it was just to indicate that the integer value returned from the method actually refers to a memory address.